<launch>
<node pkg="robo_vision" type="read_camera" name="read_usbcamera" output="screen">
    <param name="image_topic" value="/usbcamera/image"/>
    <param name="dev_name" value="/dev/ttyVideo0"/>
    <param name="image_width" value="640"/>
    <param name="image_height" value="480"/>
    <param name="exposure_time" value="56"/>
</node>
<node pkg="robo_vision" type="armor_detect" name="armor_detect" output="screen">
    <param name="intrinsic_file" value="/home/ubuntu/robot/src/robo_vision/param/camera-02-640.xml"/>
    <param name="offset_anlge_x" value="0.0"/>
    <param name="offset_anlge_y" value="-0.7"/>
</node>


</launch>
